Rubberduck.Refactorings/ReorderParameters/ReorderParametersModel.cs

```diff
@@ -44,7 +43,7 @@ private void LoadParameters()
         {
             if (TargetDeclaration == null) { return; }
 
-            Parameters = ((IParameterizedDeclaration) TargetDeclaration).Parameters.Select((s, i) => new Parameter((ParameterDeclaration)s, i)).ToList();
+            Parameters = ((IParameterizedDeclaration) TargetDeclaration).Parameters.Select((param, idx) => new Parameter(param, idx)).ToList();
```

Comment:
I wonder whether that can be simplified to something like Java's `Parameter::new`